### PR TITLE
Master performance configurator apply gdi

### PR DESCRIPTION
**Pull request by Guillaume-gdi**
This PR
- adds loggs to monitor the performance of the GPT calls during the apply of the configurator.
- changes the timeout of the OLG call to 20s.

Related to task-3248852

### CODE REVIEW

**Comment by rdeodoo** [comment]
We will change OLG timeout to 20 seconds in this PR

**Comment by Guillaume-gdi** [comment]
@rdeodoo This PR is ready for your review :heavy_check_mark:

File: addons/website/models/website.py [hunk 1]
```diff
@@ -654,6 +656,8 @@ def _render_snippet(key):
                 for key in generated_content:
                     if response.get(key):
                         generated_content[key] = (name_replace_parser.sub(website.name, response[key], 0))
+                gpt_process_time = time.time() - start_time
+                logger.info("GPT process time: %s seconds", gpt_process_time)
```
Comment (Guillaume-gdi) [inline addons/website/models/website.py:660]: I would be interested to know if we have a analysis tool to monitor this on the saas :thinking:

Comment (rdeodoo) [reply addons/website/models/website.py:660]: Not that I know of.
We can just ask for some logs here and there but we don't have any access to the logs ourselves, let alone some monitoring tools. At least from what I know.
Could be a thing tho, the runbot team has some graph about perfs based on test logs AFAIK.

Comment (rdeodoo) [inline addons/website/models/website.py:660]:
```suggestion
                logger.info("GPT process time: %s seconds", time.time() - start_time)
```

**Review by rdeodoo — CHANGES_REQUESTED** [review]
The two `time` lines are 100 lines apart, one is named to hint about `gpt` (`gpt_process_time`), not the other one (just `start_time`).

Don't we just want to track the time of the call to IAP itself actually?
If so, we can just do
```py
logger.info("GPT process time: %s seconds", response.elapsed.total_seconds())
```

Not such occurrences in the code yet but seems to be exactly what we need. If not, I would at least only time that IAP call line.

What do you think?

**Review by rdeodoo — CHANGES_REQUESTED** [review]
@Guillaume-gdi Let's modify IAP then. 
I confirmed before asking you to make the change. https://discord.com/channels/678381219515465750/687337958662995977/1177555228401356860
1. It will be a bit a nicer diff as we have response object to read elapsed property
2. It's always a valuable information to have anyway, not only for website IAP calls

**Review by rdeodoo — APPROVED** [review]
@robodoo r+ rebase-ff

**Pull request closed without merging**